solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -72,6 +73,7 @@ contract LiquidityProtection is TokenHandler, Utils, Owned, ReentrancyGuard, Tim
 
     ILiquidityProtectionSettings public immutable settings;
     ILiquidityProtectionStore public immutable store;
+    ILiquidityProtectionStats public immutable stats;
```

Comment:
Can you add an interface for LP returning both `store` and `stats`?

Reply:
Yes, but why? We're not using these functions on-chain.

Reply:
We need them for LM... these two getters and `addLiquidityFor`

Reply:
Done

Reply:
Can you add the interface for LP itself (with `store`, `stats`, and `addLiquidityFor`)? You have only added an interface for the stats themselves

Reply:
Done.